navigation/create_control/src/create_control/create_control_node.cpp

```diff
@@ -0,0 +1,19 @@
+#include "create_hw_interface.hpp"
+
+int main(int argc, char *argv[]) {
+
+  ros::init(argc, argv, "create_hw_interface");
+  ros::AsyncSpinner spinner(1);
+  spinner.start();
+  ros::NodeHandle nh;
+  Create2Interface robot(nh);
+  controller_manager::ControllerManager cm(&robot, nh);
+
+  ros::Rate r(100);
```

Comment:
Create a node parameter to change this rate.

Reply:
done